app/javascript/mastodon/actions/compose.js

```diff
@@ -301,11 +301,17 @@ export function uploadCompose(files) {
     const uploadLimit = getState().getIn(['server', 'server', 'configuration', 'statuses', 'max_media_attachments']);
     const media = getState().getIn(['compose', 'media_attachments']);
     const pending = getState().getIn(['compose', 'pending_media_attachments']);
+    const serverConfiguration = getState().getIn(['server', 'server', 'configuration']);
+    const maxMediaAttachments = serverConfiguration.getIn(['statuses', 'max_media_attachments']);
```

Comment:
Is this not the same as `uploadLimit` above?

(FWIW, I prefer the name `maxMediaAttachments` as I think it is clearer, so maybe `uploadLimit` could be renamed.)